record_web/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   flutter_web_plugins:
     sdk: flutter
 
-  web: ^0.5.1
+  web: ^1.0.0
```

Comment:
I do prefer ">=0.5.1 <2.0.0" to not break lower bound. 0.5.1 should be compatible with 1.0.0 as far as I know.

Reply:
Soon as you enable web:^1.0.0, first error you will get is:

```
Target dart2js failed: ProcessException: Process exited abnormally with exit code 1:
../../../../AppData/Local/Pub/Cache/hosted/pub.dev/record_web-1.1.1/lib/js/js_import_library.dart:26:21:
Error: A value of type 'String' can't be assigned to a variable of type 'JSAny'.
      ..innerHTML = content;
                    ^
Error: Compilation failed.
```

This patches it for 1.0.0 stable on so package does not break.

Reply:
Hmm, OK